app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -41,7 +63,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return binding.getRoot();
     }
 
+    @Override
+    public void onStop() {
+        super.onStop();
+        compositeDisposable.dispose();
+    }
+
     private void initView() {
         //
     }
+
+    private void renderContributors(List<Contributor> contributors) {
+        // TODO render contributors
```

Comment:
I'm kinda lost here. Could you explain more? You want me to create `ContributorViewModel` for each grid. And then handle click event there? Maybe need to remove [callback](https://github.com/DroidKaigi/conference-app-2017/blob/master/app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java#L12)?